content/ko/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -18,48 +18,45 @@
         <div class="row">
 
             <div class="col-md-8">
-                <h3>Objectives</h3>
+                <h3>목표</h3>
                 <ul>
-                    <li>Learn about application Deployments.</li>
-                    <li>Deploy your first app on Kubernetes with kubectl.</li>
+                    <li>애플리케이션 디플로이먼트(Deployment)에 대해 배운다.</li>
+                    <li>kubectl로 첫 애플리케이션을 쿠버네티스에 배포한다.</li>
                 </ul>
             </div>
 
             <div class="col-md-8">
-                <h3>Kubernetes Deployments</h3>
+                <h3>쿠버네티스 디플로이먼트</h3>
                 <p>
-                    Once you have a running Kubernetes cluster, you can deploy your containerized applications on top of
-                    it.
-                    To do so, you create a Kubernetes <b>Deployment</b> configuration. The Deployment instructs
-                    Kubernetes
-                    how to create and update instances of your application. Once you've created a Deployment, the
-                    Kubernetes
-                    master schedules mentioned application instances onto individual Nodes in the cluster.
+                    일단 쿠버네티스 클러스터를 구동시키면, 그 위에 컨테이너화된 애플리케이션을 배포할 수 있다.
+                    그러기 위해서, 쿠버네티스 <b>디플로이먼트</b> 설정을 만들어야 한다. 디플로이먼트는 쿠버네티스가
+                    애플리케이션의 인스턴스를 어떻게 생성하고 업데이트해야하는지를 지시한다. 디플로이먼트가 만들어지면,
+                    쿠버네티스 마스터가 해당 애플리케이션 인스턴스를 클러스터의 개별 노드에 스케줄한다.
                 </p>
 
-                <p>Once the application instances are created, a Kubernetes Deployment Controller continuously monitors
-                    those instances. If the Node hosting an instance goes down or is deleted, the Deployment controller
-                    replaces it. <b>This provides a self-healing mechanism to address machine failure or
-                        maintenance.</b></p>
+                <p>애플리케이션 인스턴스가 만들어지면, 쿠버네티스 디플로이먼트 컨트롤러는 지속적으로 이들 인스턴스를
+                    모니터링한다. 인스턴스가 구동되는 노드가 다운되거나 삭제되면, 디플로이먼트 컨트롤러가 인스턴스를
```

Comment:
근거를 조금 더 설명 부탁드립니다.

Reply:
@gochist,@ClaudiaJKang 제 의견 달아도 될지는 모르겠는데.
If the Node hosting an instance goes down or is deleted...
--> 인스턴스를 호스트하는(보관하고 있는) 노드가 다운되거나 삭제되어지면...
여기서 hosting an instance 가 Node를 꾸며주는 형태인지라 이렇게 해석함이 어떨지 의견 남김니다. 참고로 host의 의미는 "Store (a website or other data) on a server or other computer so that it can be accessed over the Internet."을 근거로 했습니다.<출처 : https://en.oxforddictionaries.com/definition/host>

Reply:
@gochist 조사 '가'가 두번 언급이 되어 '의'로 추천을 드렸습니다. 다시 읽어보니 의역이 된 것 같습니다. 의견 주셔서 감사합니다. 
해당 부분에 관해서 @zer0big 님의 리뷰를 수용해서 `인스턴스를 구동하는 노드가 다운되거나 삭제되면`은 어떠신가요?